setup.py

```diff
@@ -44,20 +43,22 @@
     "Intended Audience :: Developers",
     "Intended Audience :: System Administrators",
     "License :: OSI Approved :: Zope Public License",
-    "Programming Language :: Python",
+    "Programming Language :: Python :: 2 :: Only",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: Implementation :: CPython",
     "Topic :: Software Development",
     "Topic :: Utilities",
     ]
 
 setup(name="tl.eggdeps",
-      version="0.5dev",
+      version="0.5.dev0",
       description=__doc__.strip(),
       long_description=longdesc,
       keywords="egg eggs dependencies dependency graph tree",
       classifiers=classifiers,
       author="Thomas Lotze",
       author_email="thomas@thomas-lotze.de",
-      url="http://thomas-lotze.de/en/software/eggdeps/",
```

Comment:
Hier würde ich gern die alte URL beibehalten, zumindest bis ich mir mal angesehen habe, ob ich eine Beispielgrafik auf der Projektseite im Github einbinden kann und wie das ggf. am sinnvollsten geht. Gibt's eigentlich mehr oder weniger wohldefinierte Richtlinien dafür, was als URL üblich oder gemeinhin akzeptiert ist?

Reply:
In den Zopefoundation- und gocept-Repos verwenden wir immer die Repos-URL, weil man vom Repos aus alle relevanten Informationen finden kann und diese immer auf dem aktuellen Stand sind.